src/components/ActionList/ActionList.scss

```diff
@@ -52,7 +52,6 @@ $item-vertical-padding: ($item-min-height - line-height(body)) / 2;
   cursor: pointer;
   padding: $item-vertical-padding spacing(tight);
   border-radius: var(--p-border-radius-base);
-  border-top: 1px solid transparent var(--p-surface); // 1px gap between elements
```

Comment:
`border-top: 1px solid transparent; // 1px gap between elements` is probably still wanted here

Reply:
👋🏽 hi hi! When the `border-top` is added, it looks like this: 

![WITH transparent border top](https://user-images.githubusercontent.com/43223543/107431084-f9793700-6af3-11eb-998c-6632f680ab4d.jpg)


When it should look like this: 

![withOUT transparent border top](https://user-images.githubusercontent.com/43223543/107431169-157cd880-6af4-11eb-8e25-0c153cd3260b.jpg)

Reply:
I see, we need the border top so that the elements have a gap between them and does not cause the mouse to flicker when scrolling. This is only needed when outsidte of high contrast. If we can override it later that would be best.

Reply:
oooh hi hi! Sorry - just got around to this. Added a `margin` and I think that would fix this!

Reply:
This is what it looks like with the added border: 
![ActionLIst](https://user-images.githubusercontent.com/43223543/107813532-8a435300-6d3e-11eb-9eb5-7ecea7878ac2.jpg)

focused state:
![ActionListFocused](https://user-images.githubusercontent.com/43223543/107813609-a5ae5e00-6d3e-11eb-8936-46fd68336801.jpg)